Library/Homebrew/bintray.rb

```diff
@@ -59,19 +68,25 @@ def upload(local_file, repo:, package:, version:, remote_file:, sha256: nil, war
     args = ["--upload-file", local_file]
     args += ["--header", "X-Checksum-Sha2: #{sha256}"] unless sha256.blank?
     args << "--fail" unless warn_on_error
-    result = open_api(url, *args)
+
+    result = open_api(url, T.unsafe(*T.unsafe(args)))
```

Comment:
This may now type-check, but just by looking at it it seems wrong. A method cannot return a splat, so whatever is passed to `open_api` is probably an array and not a splat anymore.

Reply:
Just checked, you cannot even call `T.unsafe` with multiple arguments, i.e.

```ruby
T.unsafe(*[1, 2])
```

throws an `ArgumentError`.